### PR TITLE
Hello Microverse project Pull Request #2

### PR DESCRIPTION
**Pull request by sinruidotec**
**In this project i:**

- Added an HTML file with h1 and a class.
- Added a CSS file to stylize the h1 class with the color red.
- Added .gitignore file with node_modules and test.md files inside.
- Added linters.yml.

HTML and CSS project, the main task is to add the necessary files and folders to code with good practices.

.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,168 @@
-# Hello_Microverse1
-second test Microverse project
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="" alt="C:\Users\Jonathan\Desktop\second_test\Hello_Microverse1" width="140"  height="auto" />
```
Comment (Shedrack-Sunday) [inline README.md:4]: - [ ] It's nice to add your customize photo 👍 . But when you do so, kindly make sure it renders perfectly.
- Presently, the photo does not display properly. Kindly fix the path error, or delete it to make your work look clean and professional 👍

File: README.md [hunk 2]
```diff
@@ -1,2 +1,168 @@
...
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+  </ul>
+<ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
```
Comment (Shedrack-Sunday) [inline README.md:56]: - [ ] All sections with the `>` notation your code serve as a guide and should not be committed to your codebase
- This ensures you keep your work neat and proffesional 👍

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @sinruidotec    👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done extremely well so far in this project,  However, there's one adjustment you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️  
- Kudus to hitting most of the project requiremets ✔️  
  
# Required Changes  🌵 

- [x] You made some impressive implementations which I must say is a job well done. However, there are a few  I suggest   you make In line with the requirements 
- [x] Though you tried in your pull request summary, kindly make it a littlee bit more descriptive just as it was pointed out in the [Proffesional repo rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md). Your PR should state all chnages made in this branch, which will serve as a refeneec incase you want to retarct chnages or while working with a team 👍 

## Sample PR to be used:
image

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by emmiiorji — APPROVED** [review]
Hi @sinruidotec, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align='center'>

</div>

## Good Highlights✔️

- Repo is professional
- Requested changes have been implemented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**